packages/gen-ai-hub/src/orchestration/orchestration-client.ts

```diff
@@ -39,22 +39,24 @@ export class GenAiHubClient {
 export function constructCompletionPostRequest(
   input: GenAiHubCompletionParameters
 ): CompletionPostRequest {
-  return {
-    // TODO: Properly implement input params
-    input_params: {},
+  const result: CompletionPostRequest = {
     orchestration_config: {
       module_configurations: {
         templating_module_config: {
           template: input.prompt.template
         },
-        llm_module_config: input.llmConfig
-      },
-      ...(input.prompt.template_params && {
-        input_params: input.prompt.template_params
-      }),
-      ...(input.prompt.messages_history && {
-        messages_history: input.prompt.messages_history
-      })
-    }
+        llm_module_config: input.llmConfig,
+        ...(input?.filterConfig && {
+          filtering_module_config: input.filterConfig
+        })
+      }
+    },
+    input_params: input.prompt.template_params
+      ? input.prompt.template_params
+      : {},
```

Comment:
why change this? `input_params` are still optional, right?

Reply:
In the generated client it is no longer optional, but in the spec it is [still optional](https://github.tools.sap/AI/llm-orchestration/blob/main/src/spec/api.yaml#L66).
cc @tomfrenken

Reply:
I have regenerated the orchestration client.